website/docusaurus.config.js

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable */
+const versions = require('./versions.json');
+
+module.exports = {
+  title: 'Graphback',
+  tagline: 'GraphQL API for your client and server',
+  url: 'https://graphback.dev',
+  baseUrl: '/',
+  favicon: 'img/favicon.ico',
+  organizationName: 'aerogear', // Usually your GitHub org/user name.
+  projectName: 'graphback', // Usually your repo name.
+  themeConfig: {
+    disableDarkMode: true,
```

Comment:
is this incompatible with the new design?

Reply:
No, it is compatible, just needs some work in terms of styling. So I'll work on that next, possibly in another pr